src/blog/v8-release-84.md

```diff
@@ -0,0 +1,88 @@
+
+### Improved start-up time
+
+WebAssembly's baseline compiler ([Liftoff](https://v8.dev/blog/liftoff)) now supports [atomic instructions](https://github.com/WebAssembly/threads) and [bulk memory operations](https://github.com/WebAssembly/bulk-memory-operations). This means that even if you use these pretty recent spec additions, you get blazingly fast start-up times.
+
+### Better debugging
+
+In an ongoing effort to improve the debugging experience in WebAssembly, we are now able to inspect any WebAssembly frame that is live whenever you pause execution or reach a breakpoint.
+This was realized by re-using [Liftoff](https://v8.dev/blog/liftoff) for debugging. In the past, all code that had breakpoints or was stepped through needed to execute in the WebAssembly interpreter, which slowed down execution substantially (often around 100x). With Liftoff, you only lose about one third of your performance, but you can step through all code and inspect it at any time.
+
+### SIMD Origin Trial
+
+The SIMD proposal enables WebAssembly to take advantage of commonly available hardware vector instructions to accelerate compute intensive workloads. V8 has [support](https://v8.dev/features/simd) for the [WebAssembly SIMD proposal](https://github.com/WebAssembly/simd). To enable this in Chrome, use the flag `chrome://flags/#enable-webassembly-simd` or sign up for an [origin trial](https://developers.chrome.com/origintrials/#/view_trial/-4708513410415853567). [Origin trials](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md) allow developers to experiment with a feature before it is standardized, and provide valuable feedback. Once an origin has opted into the trial users are opted into the feature for the duration of the trial period without having to update Chrome flags.
+
+## JavaScript
+
+### Weak references and finalizers
+
+:::note
+**Warning!** Weak references and finalizers are advanced features! They depend on garbage collection behavior. Garbage collection is non-deterministic and may not occur at all.
+:::
+
+JavaScript is a garbage collected language, which means memory occupied by objects that are no longer reachable by the program may be automatically reclaimed when the garbage collector runs. With the exception of references in `WeakMap` and `WeakSet`, all references in JavaScript are strong and prevent the referenced object from being garbage collected. For instance,
+
+```javascript
+let globalRef = { callback: function() { console.log(“foo”); } };
+// As long as globalRef is reachable through the global scope,
+// neither it nor the function in its callback property will be collected.
+```
+
+JavaScript programmers can now hold on to objects weakly via the `WeakRef` feature. Objects that are referenced by weak references do not prevent their being garbage collected if they are not also strongly referenced.
+
+```javascript
+const globalWeakRef = new WeakRef({ callback: function() { console.log(“foo”); });
```

Comment:
Let's split this up, it's a bit too long and causes a horizontal scroll even on wide monitors.

```suggestion
const globalWeakRef = new WeakRef({
  callback: function() { console.log('foo');
});

```